.gitignore

```diff
@@ -9,9 +9,6 @@
 /src/locales/**/pseudo.po
 
 # generated graphql types
-# TODO: after apollo migration is finished, simplify gitignore
-/src/**/__generated__/*
```

Comment:
Are there no other generated files that need to be ignored?

Reply:
Nope, we only generate the types-and-hooks files and the schema file